packages/cli-exec/src/commands/exec/stop.js

```diff
@@ -15,14 +15,16 @@ export class Stop extends Command {
 
   async run() {
     let { port } = this.flags;
+    let stop = `http://localhost:${port}/percy/stop`;
+    let ping = `http://localhost:${port}/percy/healthcheck`;
```

Comment:
Should these be set with `PERCY_SERVER_ADDRESS`? 🤔 is that configurable?

Reply:
That variable is meant for the SDKs and is set by the `exec` command (not `exec:start` or others). When set by `exec`, it actually does end up as `http://localhost:${port}` (same as here) via the port flag being passed to core which doesn't allow changing the hostname and just uses `localhost`.

That variable should only be manually set if the SDK is behind a different IP address than the CLI.  You could argue that it'd be nice to `ping` and `stop` from other IPs, but then I'd argue that it's probably an additional feature to come in another PR. If we did want to implement that feature, we'd have to reconcile on how to handle both the env var and the flag (e.g. should we warn when both exist or silently ignore `--port`?).